lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -37,14 +37,7 @@ class EngineLineMetrics implements ui.LineMetrics {
     required this.widthWithTrailingSpaces,
     required this.left,
     required this.lineNumber,
-  })  : assert(displayText != null), // ignore: unnecessary_null_comparison
```

Comment:
why removed? these were for clients that are not non-null safe yet.

Reply:
Added back